src/CabanaPD_Input.hpp

```diff
@@ -110,10 +113,16 @@ class Inputs
         else if ( inputs.contains( "low_corner" ) &&
                   ( inputs.contains( "high_corner" ) ) )
         {
+            auto size = inputs["low_corner"]["value"].size();
+            if ( size != 3 )
+                log_err( std::cout, "CabanaPD requires 3d (low_corner)." );
+            if ( inputs["high_corner"]["value"].size() != 3 )
+                log_err( std::cout, "CabanaPD requires 3d (high_corner)." );
+
             auto low_corner = inputs["low_corner"]["value"];
             auto high_corner = inputs["high_corner"]["value"];
             std::string size_unit = inputs["low_corner"]["unit"];
```

Comment:
"string size_unit" is confusing here, because "size" is the array dimension not the domain size.

Reply:
Changed